elasticsearch/example/kustomization.yaml

```diff
@@ -11,10 +11,22 @@ commonAnnotations:
   "app.uw.systems/tier": "tier_4"
 
 patches:
+  # Scaling the nodes down to 1
```

Comment:
could you explain why we might want this? Looking at https://github.com/utilitywarehouse/kubernetes-manifests/pull/83401 I'm not sure what was incorrect about the previous approach

Reply:
Do you find [this comment](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/17/commits/6d98363fd9032d79bcc8881dfb7fe8fb6a98f083) more helpful? I'm not sure what else add here